sweep/sweeper.go

```diff
@@ -1752,8 +1802,10 @@ func (s *UtxoSweeper) handleBumpEvent(r *bumpResp) error {
 	case TxReplaced:
 		return s.handleBumpEventTxReplaced(r)
 
+	// There's a fatal error in creating or publishing the tx, we will
```

Comment:
just "creating" and not "or publishing" right? cause we only ever set the Event=TxErr in `handleInitialBroadcast` afaict (at least in this PR)

Reply:
`handleInitialBroadcast` does also broadcast it, but technically it should fail prior when calling `CheckMempoolAcceptance`

Reply:
> just "creating" and not "or publishing" right? cause we only ever set the Event=TxErr in

Correct, updated the comments - atm we only mark it as fatal in the initial tx creation.